tap/tracker.py

```diff
@@ -159,6 +179,15 @@ def _write_tap_version(self, filename):
         if ENABLE_VERSION_13:
             print('TAP version 13', file=filename)
 
+    def _write_plan(self, stream):
+        """
```

Comment:
This project adheres to [PEP 257](https://www.python.org/dev/peps/pep-0257/) for docstring structure. For example:
```python
"""Make imperative sentence statement.

Provide some details about what the method does.
"""
```